app/views/posts/index.html.erb

```diff
@@ -1,2 +1,21 @@
-<h1>Here is a list of posts for a given user</h1>
-<p>Find me in app/views/posts/index.html.erb</p>
+<%= render partial: "shared/userbar", locals: {
+    photo: @user.photo,
+    username: @user.name,
+    post_number: @user.posts_counter
+  } 
+%>
+<% @posts.each do |post| %>
+  <article class="post-container">
+      <h3 class="title"><%= post.title %></h3>
+      <p class="text"><%= post.text %></p>
+      <div class="comment-like-container">
+          <p>Comments: <%= post.comments_counter %></p>
+          <p>Likes:  <%= post.likes_counter %></p>
+      </div>
+  </article>
+  <article class="post-container">
+      <% post.more_recent_comments.each do |comment| %>
+          <p><%=@users.find(comment.author_id).name %> : <%= comment.text %></p>
+      <%end%>
+  </article>
+<%end%>
```

Comment:
- [ ] Clicking on each user takes you to their show page as shown in the [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_page.png?token=ANIPACIRNXBW3VN5L6SQGRLBSU4WG)